README.md

```diff
@@ -474,14 +474,14 @@ via a login protector if the operating system is reinstalled or if the disk is
 connected to another system** -- even if the new system uses the same login
 passphrase for the user.
 
-Because of this, `fscrypt encrypt` will offer to generate a recovery passphrase
-when creating a login passphrase-protected directory on a non-root filesystem.
-The recovery passphrase is simply a `custom_passphrase` protector with a
-randomly generated high-entropy passphrase.  It is strongly recommended to
-accept the prompt to generate the recovery passphrase, then store the recovery
-passphrase in a secure location.  Then, if ever needed, you can use `fscrypt
-unlock` to unlock the directory with the recovery passphrase (by choosing the
-recovery protector instead of the login protector).
+Because of this, `fscrypt encrypt` will automatically generate a recovery
+passphrase when creating a login passphrase-protected directory on a non-root
+filesystem (unless the `--no-recovery` option is given).  The recovery
+passphrase is simply a `custom_passphrase` protector with a randomly generated
+high-entropy passphrase.  It is strongly recommended to record the recovery
+passpshrase in a secure location, and not disable it.  Then, if ever needed, you
```

Comment:
> It is strongly recommended to record the recovery passpshrase

There is a typo in `passpshrase`.

> It is strongly recommended to record the recovery passphrase in a secure location. **Period.**

Humans need to be guided and slightly forced to do the right thing indeed, but _"and not disable it"_ is a bit verbose. It is implied because it makes logical sense. The reason we are being very explicit about the importance of the recovery passphrase is to bring it to the users' attention and to convince them not to be lazy and skip this step. But once it is brought to their attention, I think it's save to assume they understand that the protector shouldn't be removed when used.

Reply:
> I think it's save to assume they understand that the protector shouldn't be removed when used.

If you want to be really really careful, you can add a warning to the `fscrypt metadata remove-protector-from-policy` command.

> fscrypt metadata remove-protector-from-policy (...)
>
> Warning: The protector about to be removed is a passphrase_protector. You may lose your recovery passphrase. There may be no more ways to recover your data when you lose access to your root partition. Are you sure you want to continue? y/n

Reply:
I would change that part:  "It is strongly recommended to record the recovery passphrase in a secure location, and not disable it"
To:
Password is stored in encrypted folder so to be able to use it you  MUST store it in pass manager or other save place

Also i would change "unless the --no-recovery option is given" to 

--no-recovery option will disable generating recovery protector an pass to it so if you are prepared for DATA LOSS you can use it if you need to bypass generating recovery for whatever reason

And place it at the end of this paragraph

Reply:
> There is a typo in passpshrase.

Fixed.

> Humans need to be guided and slightly forced to do the right thing indeed, but "and not disable it" is a bit verbose.

Removed that part.

> If you want to be really really careful, you can add a warning to the fscrypt metadata remove-protector-from-policy command.

`fscrypt metadata` needs a lot of usability improvements in general.  I'm not planning to address this right now, as `fscrypt metadata` isn't commonly used.  And if someone is using it to remove a recovery protector, they probably got there through `fscrypt_recovery_readme.txt` which explains the recovery passphrase and recommends against removing it.

> I would change that part: "It is strongly recommended to record the recovery passphrase in a secure location, and not disable it"
> To:
> Password is stored in encrypted folder so to be able to use it you MUST store it in pass manager or other save place

I did: "Initially, the recovery passphrase is stored in a file in the encrypted directory itself; therefore, to use it you **must** record it in another secure location.  It is strongly recommended to do this."

> Also i would change "unless the --no-recovery option is given" to
>
> --no-recovery option will disable generating recovery protector an pass to it so if you are prepared for DATA LOSS you can use it if you need to bypass generating recovery for whatever reason

I did: "If you really want to disable the generation of a recovery passphrase, use the `--no-recovery` option.  Only do this if you really know what you are doing and are prepared for potential data loss."